python_modules/dagster/dagster/utils/__init__.py

```diff
@@ -27,7 +27,6 @@
 from dagster.seven import IS_WINDOWS, multiprocessing
 from dagster.seven.abc import Mapping
 
-from .alert import make_email_on_pipeline_failure_sensor, make_email_on_run_failure_sensor
```

Comment:
I think we should keep these, o.w. our examples will be wrong (where we're importing from `dagster.utils`).  Also, we might break folks who've imported following our docs.

Reply:
I guess to avoid the circular import scenario, you would want to leave stubs that lazily import from `alert.py` and forward the args along?

Reply:
in theory I changed the examples, but fair enough on the people who imported it...